templates/template.scss.njk

```diff
@@ -12,22 +12,22 @@
     {%- set ttfIndex = formats.indexOf('ttf') -%}
     {%- set svgIndex = formats.indexOf('svg') %}
     src: {% if eotIndex != -1 -%}
-            url("{{ fontPath }}{{ fontName }}.eot?#iefix") format("embedded-opentype")
+            locale({{ fontName }}) url("{{ fontPath }}{{ fontName }}.eot?#iefix") format("embedded-opentype")
```

Comment:
@abhi05b 

Thank you for contributing with us.
What would be the purpose of adding `locale` to the SCSS template?